docs/EVMC.md

```diff
@@ -0,0 +1,50 @@
+# EVMC – Ethereum Client-VM Connector API {#mainpage}
+
+The EVMC is the low-level ABI between Ethereum Virtual Machines (EVMs) and 
+Ethereum Clients. On the EVM-side it supports classic EVM1 and [Ewasm].
+On the Client-side it defines the interface for EVM implementations 
+to access Ethereum environment and state.
+
+
+## Modules
+
+- [EVMC](@ref EVMC) 
+   – the main component that defines API for VMs and Clients (Hosts).
+- [EVMC Loader](@ref loader)
+   – the library for loading VMs implemented as Dynamically Loaded Libraries (DLLs, shared objects). 
+- [EVMC Helpers](@ref helpers)
+   – a collection of utility function for easier integration with EVMC.
+- [EVM Instructions](@ref instructions) 
+   – the library with collection of metrics for EVM1 instruction set.
+
+
+[Ewasm]: https://github.com/ewasm/design#ethereum-flavored-webassembly-ewasm-design
```

Comment:
https://github.com/ewasm/design#what-is-ethereum-flavored-webassembly-ewasm

Reply:
Why does it need to link to the heading?